pow/src/main/java/tech/pegasys/artemis/pow/DepositContractListener.java

```diff
@@ -88,7 +88,14 @@ public void stop() {
                 throw new Eth1RequestException(
                     "Eth1 call has failed:" + ethCall.getError().getMessage());
               } else {
-                return ethCall.getValue();
+                String value = ethCall.getValue();
+                if (value == null) {
+                  throw new Eth1RequestException(
+                      "Eth1 call on state of block number "
+                          + blockHeight
+                          + " seems to be outdated on the node cache.");
```

Comment:
What about "Eth1 call has failed:  data at block number X is unavailable."

Reply:
I like that. Done.